spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc

```diff
@@ -75,162 +71,154 @@ SPRING_APPLICATION_JSON='{
+Data Flow lets you debug the deployed applications.
+This is accomplished by enabling the remote debugging feature of the JVM through deployment properites, as shown in the following example:
 
-For example:
 [source,bash]
 ----
 stream deploy --name mystream --properties "deployer.fooApp.local.debugPort=9999"
 ----
-The above will start the `fooApp` application in debug mode allowing remote debugger to be attached on port 9999. 
-By default the application will start in a ’suspend’ mode waiting for the remote debug session to be attached (started). Otherwise you can  provide an additional `debugSuspend` property with value `n`.
 
-Also, in the event when there is more then one instance of the application, the debug port for each instance will be the value of `debugPort` + instanceId.
+The preceding example starts the `fooApp` application in debug mode, allowing a remote debugger to be attached on port 9999.
+By default, the application starts in a ’suspend’ mode and waits for the remote debug session to be attached (started). Otherwise, you can  provide an additional `debugSuspend` property with value `n`.
+
+Also, when there is more then one instance of the application, the debug port for each instance is the value of `debugPort` + instanceId.
+
+NOTE: Unlike other properties you must NOT use a wildcard for the application name, since each application must use a unique debug port.
 
-NOTE: Unlike other properties you must NOT use wild-card for the application name, since each application must use a unique debug port.
 
 
-==== Log redirect
+==== Log Redirect
 
-Given that each application is a separate process with each maintaining it's own set of logs, accessing individual logs could be a bit inconvinient especially in the early stages of the development when logs are accessed more often (i.e., debugging, troubleshooting, etc.).
-Since it is also a common pattern to rely on Local SCDF Server which deploys each application as a local JVM process using link:https://github.com/spring-cloud/spring-cloud-deployer-local[local-deployer], the framework provides support for redirecting such logs to a parent process’s streams (both stdout and stderr). So essentially with Local SCDF Server the application logs will appear in the logs of running Local SCDF Server.
+Given that each application is a separate process that maintains its own set of logs, accessing individual logs could be a bit inconvenient, especially in the early stages of development, when logs are accessed more often.
+Since it is also a common pattern to rely on a local SCDF Server that deploys each application as a local JVM process with link:https://github.com/spring-cloud/spring-cloud-deployer-local[local-deployer], the framework provides support for redirecting such logs to a parent process’s streams (both stdout and stderr). Thus, with a local SCDF Server, the application logs appear in the logs of the running local SCDF Server.
```

Comment:
"process’s streams (both stdout and stderr)" -> perhaps: "process’s logs (both stdout and stderr)"

Reply:
cleaned it up